CluedIn.Connector.SqlServer.sln

```diff
@@ -52,6 +52,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Connector.SqlServer", "src\
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Connector.SqlServer.Unit.Tests", "test\unit\Connector.SqlServer.Test\Connector.SqlServer.Unit.Tests.csproj", "{5F2D36E4-2B4A-47EF-B4FC-703FFBB074E4}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Core", "..\CluedIn\Code\Core\Core.csproj", "{16946CAE-528E-4C2F-8E42-0DBAEFCE5557}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Core.Agent", "..\CluedIn\Code\Core.Agent\Core.Agent.csproj", "{68366B00-88D3-4063-91B2-ECEBD9FB2CAC}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Crawling", "..\CluedIn\Code\Crawling\Crawling.csproj", "{26C11789-BA1C-4C6A-BFD7-AEE3BBA885CE}"
+EndProject
 Global
```

Comment:
This is from debugging right? They should be removed

Reply:
Reviewed too early lol. Was on draft. These have been reverted